### DISCUSSION TITLE
HVDC Models for DC Line Element

### DISCUSSION THREAD

**Discussion by victormbarriosb** [e2nIEE/pandapower#1077] [Ideas]
It would be great if the **DC line element** contained other models that are commonly used in power systems today, such as **monopolar** or **bipolar** link, maybe **homopolar** as well. Is there already any progress in that area?

**Comment by lthurner** [comment]
No progress yet as far as I know, but I agree it would be very useful. If you have the possibility to pitch in, that would be much appreciated!
